src/app/clusters/diagnostic-logs-server/diagnostic-logs-server.cpp

```diff
@@ -20,15 +20,20 @@
 #include <app/CommandHandlerInterface.h>
 #include <app/ConcreteCommandPath.h>
 #include <app/clusters/diagnostic-logs-server/diagnostic-logs-server.h>
+#include <app/server/Server.h>
 #include <app/util/af.h>
 #include <lib/support/BytesCircularBuffer.h>
+#include <lib/support/ScopedBuffer.h>
 
 #include <array>
 
+// We store our times as millisecond times, to save space.
```

Comment:
does the specs need to be updated here ; if spec says microsecond that seems overkill ...

Reply:
Spec does not have a millisecond time type... It has microseconds and seconds.

Reply:
And it's probably better to not make random spec changes here at this point.